frontend/src/pages/pipelines/global/runs/ManageRunsPage.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+
+import {
+  Bullseye,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Spinner,
+  EmptyStateHeader,
+  EmptyStateFooter,
+  EmptyStateActions,
+  Button,
+  Breadcrumb,
+  BreadcrumbItem,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon, PlusCircleIcon } from '@patternfly/react-icons';
+
+import { usePipelineActiveRunsTable } from '~/concepts/pipelines/content/tables/pipelineRun/usePipelineRunTable';
+import { PipelineRunSearchParam } from '~/concepts/pipelines/content/types';
+import {
+  experimentRunsRoute,
+  experimentsBaseRoute,
+  experimentsCompareRunsRoute,
+  experimentsCreateRunRoute,
+} from '~/routes';
+import ApplicationsPage from '~/pages/ApplicationsPage';
+import { useExperimentByParams } from '~/pages/pipelines/global/experiments/useExperimentByParams';
+import { useCompareRuns } from '~/concepts/pipelines/content/compareRuns/CompareRunsContext';
+import { getProjectDisplayName } from '~/pages/projects/utils';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import usePipelineFilter, {
+  FilterOptions,
+} from '~/concepts/pipelines/content/tables/usePipelineFilter';
+import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
+import { PipelineRunTabTitle, PipelineRunType } from './types';
+import { ManageRunsTable } from './ManageRunsTable';
+import PipelineRunVersionsContextProvider from './PipelineRunVersionsContext';
+
+interface ManageRunsPageInternalProps {
+  experiment: ExperimentKFv2;
+}
+
+export const ManageRunsPageInternal: React.FC<ManageRunsPageInternalProps> = ({ experiment }) => {
+  const navigate = useNavigate();
+  const { selectedRuns } = useCompareRuns();
+  const { namespace, project } = usePipelinesAPI();
+  const [[{ items: runs, totalSize }, loaded, error], { initialLoaded, ...tableProps }] =
+    usePipelineActiveRunsTable();
+  const filterProps = usePipelineFilter(tableProps.setFilter, {
+    [FilterOptions.EXPERIMENT]: {
+      label: experiment.display_name,
+      value: experiment.experiment_id,
+    },
+  });
+  const selectedRunIds = selectedRuns.map((selectedRun) => selectedRun.run_id);
+
+  if (error) {
+    return (
+      <Bullseye>
+        <EmptyState>
+          <EmptyStateHeader
+            titleText="There was an issue loading runs"
+            icon={<EmptyStateIcon icon={ExclamationCircleIcon} />}
+            headingLevel="h2"
+          />
+          <EmptyStateBody>{error.message}</EmptyStateBody>
+        </EmptyState>
+      </Bullseye>
+    );
+  }
+
+  if (!loaded && !initialLoaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+
+  if (loaded && totalSize === 0 && !tableProps.filter) {
```

Comment:
can you add a screenshot for this

Reply:
<img width="1156" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/96431149/1f2ac84f-8fac-450c-b408-c60bcd7f5644">

Reply:
added it to the description